src/components/MessagesList/MessagesGroup/Message/MessagePart/Mention.vue

```diff
@@ -88,6 +90,15 @@ export default {
 			return this.$store.getters.getActorType() === 'users'
 				&& this.id === this.$store.getters.getUserId()
 		},
+		isDarkTheme() {
+			return window.getComputedStyle(document.body)
+				.getPropertyValue('--background-invert-if-dark') === 'invert(100%)'
+		},
+		avatarUrl() {
+			return generateOcsUrl('apps/spreed/api/v1/room/{token}/avatar' + (this.darkTheme ? '/dark' : ''), {
+				token: this.id,
+			})
+		},
```

Comment:
Same as https://github.com/nextcloud/spreed/blob/9aed218ee1aead4c79a270fbd284d0dd4246ba91/src/components/ConversationIcon.vue#L194-L199

Also missing in https://github.com/nextcloud/spreed/pull/9332

Reply:
I think for the chat it's totally fine to see the long cached version. If the user hits ctrl+f5 it gets updated right away, otherwise it can take some time. With GitHub and other services you have the same for a much longer time?
I mean we can of course try to use it when we have the info available, but e.g. the mention list does not have the data, so we would need to cross the information based on id = token from the conversation store, adding quite some complexity, just for the rare case